Dockerfile

```diff
@@ -42,7 +42,7 @@ RUN rm /install.sh
 
 WORKDIR /srv/app
 
-COPY --chown=snyk:snyk --from=skopeo-build /usr/bin/skopeo /usr/bin/skopeo
+COPY --chown=snyk:snyk --from=skopeo-build /usr/local/bin/skopeo /usr/bin/skopeo
```

Comment:
The Build step was failing, thanks to `git blame` I found thata the path had changed -  https://github.com/containers/skopeo/commit/012ed6610ef9104b313314546b2031efa6eef6a1